WcaOnRails/app/views/teams/edit.html.erb

```diff
@@ -1,14 +1,10 @@
-<% provide(:title, "Editing team #{@team.friendly_id}") %>
+<% provide(:title, "Editing team #{@team.friendly_id} members") %>
 
 <div class="container">
   <h3><%= yield(:title) %></h3>
-
+  <%= alert :info, "This team is hidden" if @team.hidden? %>
```

Comment:
Is this interpreted as `alert(:info, "This team is hidden" if @team.hidden?)` or `alert(:info, "This team is hidden") if @team.hidden?`?
I've always found the optional parenthesis to be disturbing, but in this case it actually looks ambiguous.
(It's probably well defined by ruby, I'm just giving a reader's point of view)

Reply:
It's certainly interpreted as expected `alert(:info, "This team is hidden") if @team.hidden?`, but given you pointed it out, I'm definitely ok with adding parentheses here!